ddtrace/contrib/aiohttp/patch.py

```diff
@@ -104,10 +104,13 @@ def _patch_client(aiohttp):
 
 
 def patch():
-    # Legacy patch aiohttp_jinja2
-    from ddtrace.contrib.aiohttp_jinja2.patch import patch as aiohttp_jinja2_patch
+    try:
+        # Legacy patch aiohttp_jinja2
+        from ddtrace.contrib.aiohttp_jinja2.patch import patch as aiohttp_jinja2_patch
 
-    aiohttp_jinja2_patch()
+        aiohttp_jinja2_patch()
+    except Exception:
+        log.debug("Failed to patch aiohttp_jinja2 from aiohttp integration", exc_info=True)
```

Comment:
An additional note here on when/why this is occurring.

Reply:
it'll have the traceback, so we can see if it is an import error or something else.

or are you looking for a comment to explain that we try/except because this fails if `aiohttp_jinja2` is not available?

Reply:
Right, if `aiohttp_jinja2` is not installed, then the debug line can be ignored.

Reply:
let me add a comment 👍🏻 I am ok keeping the debug log since the behavior is meant to be the same as what we have now e.g. `aiohttp` patches `aiohttp_jinja2`. and then this all goes away in 1.0

Reply:
added a comment, let me know if that helps clarify or if we should do something else